### DISCUSSION TITLE
Welcome to InstaCracker-CLI Discussions!

### DISCUSSION THREAD

**Discussion by akhatkulov** [akhatkulov/InstaCracker-CLI#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vbudiredla** [comment]
How to get Instagram password with instacracke-cli?

**Reply by akhatkulov** [reply]
simple man, it's easy ya'know right?

**Reply by vbudiredla** [reply]
Please tell how to get Instagram password

**Reply by vbudiredla** [reply]
It's my humble request

**Reply by akhatkulov** [reply]
you can get password with bruteforce attack

**Reply by vbudiredla** [reply]
I need your help to get Instagram password

**Reply by vbudiredla** [reply]
I followed you in Instagram. My Instagram id vijju_vijay_666.please follow back me in Instagram. And i message there

**Comment by vbudiredla** [comment]
Hello my name is vijay and I want crack the instagram password with your
help.. I contact you in github application

**Reply by rohit980752** [reply]
Hello my name rohit sharma I want crack the instagram password

**Comment by vbudiredla** [comment]
Please follow back in Instagram. My Instagram id vijju vijay 666

**Reply by akhatkulov** [reply]
okay

**Comment by vbudiredla** [comment]
What are you doing?

**Comment by vbudiredla** [comment]
How to crack Instagram password please tell. Step by step

**Comment by Gulista2580** [comment]
How to crack Instagram password please tell me step by step

**Comment by LuckyNameMaster** [comment]
Hi , i need help to crack a instagram password!

**Reply by samqureshi7227-cloud** [reply]
Yes

**Comment by yahia-bel1** [comment]
Sl I installed it and then what should I type
